src/tests/.Net4.6.2/Microsoft.PowerFx.Connectors.Tests/Microsoft.PowerFx.Connectors.Tests.csproj

```diff
@@ -14,8 +14,8 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <Reference Include="System.Web" />
     
-    <PackageReference Include="Azure.Identity" Version="1.11.0" /> <!-- CVE-2024-29992 -->
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.61.1" /> <!-- CVE-2024-27086 -->
+    <PackageReference Include="Azure.Identity" Version="1.11.4" />
```

Comment:
>Identity [](http://example.com/codeflow?start=37&length=8)

is this a necessary  part of the hotfix? 

(especially considering this is a potential QFE candidate)

Reply:
Latest main has this updates. Clearly, this should not be part of the hotfix; but let's confirm it goes away when you rebase on latest.